docs/default-theme-config/README.md

```diff
@@ -226,6 +226,8 @@ module.exports = {
   themeConfig: {
     // Assumes GitHub. Can also be a full GitLab url.
     repo: 'vuejs/vuepress',
+    // Optional, your docs repo address, fallback to repo when it's not given.
```

Comment:
<code>Defaults to &#x60;themeConfig.repo&#x60;</code>

Reply:
@egoist 
Seems the same meaning

Reply:
It is clearer if u specify `themeConfig.repo`